test/images/agnhost/Dockerfile

```diff
@@ -20,15 +20,35 @@ CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
 # install necessary packages:
 # - curl, nc: used by a lot of e2e tests
 # - iproute2: includes ss used in NodePort tests
-RUN apk --update add curl netcat-openbsd iproute2 && rm -rf /var/cache/apk/*
+# from iperf image
+# install necessary packages: iperf, bash
+RUN apk --update add curl netcat-openbsd iproute2 iperf bash && rm -rf /var/cache/apk/* \
+  && ln -s /usr/bin/iperf /usr/local/bin/iperf \
+  && ls -altrh /usr/local/bin/iperf
+
+# from iperf
+RUN apt-get update && apt-get install -y --no-install-recommends iperf bash \
```

Comment:
You have both the `apt-get` and `apk` commands here; I'm a little confused by that.

Reply:
oups, I forgot to clean this up, I've included iperf and bash in the RUN above.